src/Search/SearchReducer.test.js

```diff
@@ -0,0 +1,10 @@
+import SearchReducer from './SearchReducer';
+
+describe('SearchReducer reducer', () => {
+  it('should return default state', () => {
+    const newState = SearchReducer(undefined, {});
```

Comment:
it would be nice to test reducer with action passing

it's pure function so won't be a problem

Reply:
Passed with SortBy and SearchBy cases